jwst/guider_cds/guider_cds.py

```diff
@@ -98,6 +129,72 @@ def guider_cds(model):
     return new_model
 
 
+def get_ref_arr(model, imshape, gain_model=None, readnoise_model=None):
+    """
+    Short Summary
+    -------------
```

Comment:
Remove the two lines above. It is clear this is a short summary and we use a convention where we don't actually write those two lines.